### DISCUSSION TITLE
Already brought this app

### DISCUSSION THREAD

**Discussion by alrightson** [aonez/Keka#1091] [Q&A] [answered]
Hello,

I have already brought this app, but I have had to do a fresh install of my os. When I went to re-download this app it is telling me i have to buy  it again?

**Comment by aonez** [comment, marked as answer]
I’ve seen this App Store glitch before, you can probably click on Buy and
it will let you know that you’ve already bought it. But if you want to be
sure go to your purchased apps section:

https://support.apple.com/guide/app-store/view-and-manage-purchased-apps-fir4cbb7ae76/mac
